src/app/course/challenge/course-challenge-snippet/course-challenge-snippet.component.html

```diff
@@ -1,5 +1,19 @@
 <div class="flex flex-col gap-5">
     <ng-container *ngIf="course else skeletonTemplate">
+        <div layout="column"
+             layout-align="start center"
+        layout-padding="100px">
+            <button
+                appearance="flat"
+                [routerLink]="['..']"
+                icon="tuiIconArrowLeft"
+                aria-label="Back to Homepage"
+                size="s"
+                tuiButton
+            >
+                Homepage
```

Comment:
The convention of the back navigation button names is `Back to ...`. I think here you should use `Back to Homepage`. You already have the aria-label right, they should be consistent.